src/_posts/whats-new-in-0.16.0.md

```diff
@@ -0,0 +1,128 @@
+layout: post
+title: What's new in Infection 0.16.0
+date: 2020-03-29 11:03:56
+type: '{{type}}'
+---
+
+Release: https://github.com/infection/infection/releases/tag/0.16.0
+
+## BC Breaks
+
+* Infection now requires PHP 7.3.12+. If you can't upgrade for some reason upgrade, don't worry, you can still use previous versions.
+* We had [a bug](https://github.com/infection/infection/pull/1105) with the config file loading order. Infection tried to load `infection.json.dist` file first, and then `infection.json`, while it should be the other way around. Now the order is: `infection.json`, then `infection.json.dist`.
+
+## Performance
+
+### Allow the initial test suite to be skipped (`--skip-initial-test-suite`)
+
+Infection's first step is executing your project's test suite. If it fails, further mutation process is useless, because any Mutant can be marked as Killed just because of the failing tests for the original code.
+
+That's why passing test suite is essential.
+
+If you have already run the test suite to ensure it is passing, and you are providing coverage using `--coverage` then you can use `--skip-initial-test-suite` option to increase performance by not running the entire test suite again.
+
+<p class="tip">Note that it is extremely important that you ensure the test suite is passing when using this option, otherwise test failures would appear like caught mutations and those mutations may be reported incorrectly.</p>
+
+### Other performance improvements
```

Comment:
@sanmai I left all the performance-related tickets that were done recently, if you will be able to help, here they are